labels.go

```diff
@@ -57,11 +57,60 @@ func setLoadbalancerSticky(lb *dynamic.ServersLoadBalancer, val string) error {
 	}
 
 	if v {
-		lb.Sticky = &dynamic.Sticky{}
+		if lb.Sticky == nil {
+			lb.Sticky = &dynamic.Sticky{ Cookie: &dynamic.Cookie{} }
```

Comment:
@nokjuh might be cleaner to create a function to initialise the struct and reuse it through the sticky related functions.
```go

func NewSticky() *Sticky {
    return &dynamic.Sticky{ Cookie: &dynamic.Cookie{} }
}

lb.Sticky = *NewSticky()
```

Reply:
This is now done.